tensorflow/python/ops/array_ops.py

```diff
@@ -1023,6 +1023,9 @@ def meshgrid(*args, indexing="xy", name="meshgrid"):
   Returns:
     outputs: A list of N `Tensor`s with rank N
   """
+  indexing = kwargs.get("indexing", "xy")
+  name = kwargs.get("name", "meshgrid")
```

Comment:
We should bail if there are unknown keyword args.  Not sure how best to express that.